packages/front/src/routes/association/[identifier]/Association.controller.test.js

```diff
@@ -9,12 +12,14 @@ vi.mock("$lib/store/association.store", () => ({
 }));
 
 describe("Association Controller", () => {
+    const RNA_SIREN = { rna: "RNA", siren: "SIREN", id: "QZO24DK32QKD" };
     const SIREN = "123456789";
     let controller;
 
     beforeAll(() => {
         associationService.getEstablishments.mockResolvedValue([]);
         associationService.getAssociation.mockResolvedValue({});
+        mockedRnaSirenService.getAssociatedIdentifier.mockImplementation(async () => [RNA_SIREN]);
```

Comment:
Je trouve pas lisible et ça s'affiche comme une erreur chez moi les `async` sans `await`

```suggestion
        mockedRnaSirenService.getAssociatedIdentifier.mockImplementation(() => Promise.resolve([RNA_SIREN]);
```
ou

```suggestion
        mockedRnaSirenService.getAssociatedIdentifier.mockResolvedValue([RNA_SIREN]);
```

Reply:
+ 1 pour le mockResolveValue

Reply:
je vous déteste